h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstCumu.java

```diff
@@ -37,25 +39,118 @@ public String str() {
   @Override
   public ValFrame apply(Env env, Env.StackHelp stk, AstRoot asts[]) {
     Frame f = stk.track(asts[1].exec(env)).getFrame();
-
-    if (f.numCols() != 1) throw new IllegalArgumentException("Must give a single numeric column.");
+    AstRoot axisAR = asts[2];
     if (!f.anyVec().isNumeric()) throw new IllegalArgumentException("Column must be numeric.");
```

Comment:
If you have more than 1 column, then this check may or may not fail (at random) for a dataset having non-numeric columns